graphdatascience/tests/unit/resources/example_server_endpoints.py

```diff
@@ -86,7 +86,7 @@
     "gds.alpha.sllpa.write",
     "gds.alpha.sllpa.write.estimate",
     "gds.alpha.systemMonitor",
-    "gds.alpha.triangles",
+    "gds.triangles",
```

Comment:
btw. these example_server_endpoints are just for a unit-test. You dont need to change them :)